hydro/distribution.yaml

```diff
@@ -3883,6 +3883,17 @@ repositories:
       type: git
       url: https://github.com/UC3MSocialRobots/maggie_devices_msgs.git
       version: hydro-devel
+    release:
+      packages:
+      - eyelids_msgs
+      - ir_controller_msgs
+      - maggie_devices_msgs
+      - motor_controller_msgs
+      - rfid_msgs
```

Comment:
That is the idea. What we pretend is to do generic messages for using in several different robots. Now is only maggie robot, but we are developing two different robots and trying to unify the messages for similar devices (hardware). If you still think that is better to include the prefix "maggie_", tell me and I modify the names. Thanks for your comment.